leakcanary-haha/src/main/java/leakcanary/ObjectIdMetadata.kt

```diff
@@ -3,8 +3,8 @@ package leakcanary
 import leakcanary.HprofParser.Companion.BITS_FOR_FILE_POSITION
 
 enum class ObjectIdMetadata {
-  PRIMITIVE_WRAPPER,
-  PRIMITIVE_ARRAY_OR_WRAPPER_ARRAY,
+  PRIMITIVE_WRAPPER_OR_PRIMITIVE_ARRAY,
+  PRIMITIVE_WRAPPER_ARRAY,
```

Comment:
Needed to identify PRIMITIVE_WRAPPER_ARRAY for retained size computation (because those are skipped but point to other instances) so swapped out the merge.